### PR TITLE
Bump to symfony 5.3 and use rc versions of sonata

### PR DESCRIPTION
**Pull request by franmomu**
I am targeting this branch, because is where symfony 5 is supported.

## Changelog

```markdown
### Removed
- Removed support for Symfony 5.2
```

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -21,23 +21,22 @@
...
+        "symfony/dependency-injection": "^4.4 || ^5.3",
+        "symfony/form": "^4.4 || ^5.3",
+        "symfony/http-foundation": "^4.4 || ^5.3",
+        "symfony/http-kernel": "^4.4 || ^5.3",
+        "symfony/intl": "^4.4 || ^5.3",
+        "symfony/options-resolver": "^4.4 || ^5.3",
         "twig/twig": "^2.12 || ^3.0"
     },
     "conflict": {
         "doctrine/annotations": "<1.8",
```
Comment (jordisala1991) [inline composer.json:36]: do we need this conflict?

File: composer.json [hunk 2]
```diff
@@ -54,10 +53,10 @@
         "phpstan/phpstan-symfony": "^0.12.11",
         "psalm/plugin-phpunit": "^0.15.0",
```
Comment (jordisala1991) [inline composer.json:54]: this can be ^0.16

Comment (franmomu) [reply composer.json]: done, thanks!

File: composer.json (hunk 1 above)
Comment (franmomu) [reply composer.json]: I remember there were problems with php 7.4 I think, let's see if it's not required anymore.

File: composer.json [hunk 3]
```diff
@@ -52,12 +50,12 @@
...
+        "psalm/plugin-phpunit": "^0.16",
         "psalm/plugin-symfony": "^2.0",
-        "sonata-project/doctrine-orm-admin-bundle": "^4.0@dev",
-        "symfony/framework-bundle": "^4.4 || ^5.2",
-        "symfony/phpunit-bridge": "^5.2",
-        "symfony/templating": "^4.4 || ^5.2",
+        "sonata-project/doctrine-orm-admin-bundle": "^4.0@rc",
+        "symfony/framework-bundle": "^4.4 || ^5.3",
+        "symfony/phpunit-bridge": "^5.3",
+        "symfony/templating": "^4.4 || ^5.3",
```
Comment (jordisala1991) [inline composer.json:58]: do we use this templating thing?

**Review by jordisala1991 — APPROVED** [review]

**Pull request merged**